.github/workflows/apply-applications-prd.yaml

```diff
@@ -0,0 +1,28 @@
+name: Deploy to the prd environment
+
+on:
+  push:
+    branches:
+      - 'main'
+
+jobs:
+
+  deploy:
+
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Git checkout
+        uses: actions/checkout@v1
+
+      - uses: tale/kubectl-action@v1
+        with:
+          base64-kube-config: ${{ secrets.KUBE_CONFIG_DEV }}
```

Comment:
KUBE_CONFIG_PRD 

you should create a service account with cluster-admin permissions, then generate kube-config.yaml and create a GitHub secret KUBE_CONFIG_PRD containing its content (base64 encoded)

Reply:
Renamed